packages/trackerless-network/src/logic/StreamEntryPointDiscovery.ts

```diff
@@ -138,11 +134,11 @@ export class StreamEntryPointDiscovery {
 
     async storeSelfAsEntryPointIfNecessary(
         streamPartId: StreamPartID,
-        joiningEmptyStream: boolean,
+        avoidNetworkSplit: boolean,
         entryPointsFromDht: boolean,
         currentEntrypointCount: number
     ): Promise<void> {
-        if (joiningEmptyStream) {
+        if (avoidNetworkSplit) {
             await this.storeSelfAsEntryPoint(streamPartId)
```

Comment:
This is done in both if branches, and we catching errors in only one of them? Maybe we can remove the try-catch?